pandas/core/arrays/_mixins.py

```diff
@@ -517,7 +517,9 @@ def _cast_quantile_result(self, res_values: np.ndarray) -> np.ndarray:
     # numpy-like methods
 
     @classmethod
-    def _empty(cls, shape: Shape, dtype: ExtensionDtype) -> Self:
+    def _empty(
+        cls, shape: Shape, dtype: ExtensionDtype, fill_value: object = None
```

Comment:
I don't think adding a `fill_value` argument everywhere is a desirable solution

Reply:
I understand.  What's tricky is that we have no good way to query SCALARS (defined elsewhere) as to what sort of Array they want.  In the case of complex128, it's obviously an ndarray of complex numbers.  In the case of Pint quantities, it needs to be an array of whatever types can hold the `magnitude` value.  But there's no way to ask that question from within Pandas right now.  And reading about other EA types, which could have multiple independent arrays actually backing whatever object type is facially presented to `_empty` is even more difficult.

That said, if the ExtensionDtype had an interface that could either return an obj_dtype (which would be the dtype of the underlying Array) or a callable (which could wrap whatever complexity it wants to into something that _empty could process), maybe that would be cleaner.

Should I take a crack at making ExtensionDtype a little more introspectable?  A groovy side-effect would be the efficient support of int8 and float16 EA datatypes.

Reply:
Would https://github.com/pandas-dev/pandas/pull/53089 help?

Reply:
Looks like it nibbles around the same edges.  I'll see what I can do.  If there's an obvious git/github sequence that puts me in the right spot as far as "add this PR to that PR and let me hack" please tell me.  Otherwise, I'll construct things manually (which might make merging harder later).

Reply:
Answering my own question, I've found this article on stacked PRs: https://zx77.medium.com/stacked-pull-requests-with-github-f407b5d371ea .  Is this the right way to think about the git part of making these changes on the PR #53089?

Reply:
Maybe a pull request to @jbrockmendel 's branch would be a good way to propose changes

Reply:
I could not figure out the easy way to do that (I'm the opposite of a git expert).  If you can tell me the git command for that, I can adjust.  I didn't need to modify anything they did (other than fix an assertion failure in boolean arrays that didn't like seeing "bool" types coming up from the ObjDtype).